psdk_interfaces/msg/SingleBatteryInfo.msg

```diff
@@ -0,0 +1,15 @@
+std_msgs/Header header
+
+uint8 batteryindex
+float32 voltage
+float32 current
+float32 fullcapacity
+float32 capacityremain
+float32 temperature
+uint8 cellcount
+uint32 selfcheckerror
+uint32 closedreason
+uint16 abnormalcomm
+uint16 isembed
```

Comment:
can we have all these params using camel case, so `cell_count` rather than `cellcount`?

Reply:
already changed it you can review it